bindings/src/test/java/io/pravega/storage/filesystem/FileSystemStorageTest.java

```diff
@@ -45,6 +46,8 @@ public void setUp() throws Exception {
         this.baseDir = Files.createTempDirectory("test_nfs").toFile().getAbsoluteFile();
         MetricsConfig metricsConfig = MetricsConfig.builder().with(MetricsConfig.ENABLE_STATISTICS, true).build();
         MetricsProvider.initialize(metricsConfig);
+        StatsProvider statsProvider = MetricsProvider.getMetricsProvider();
```

Comment:
Why modifying this test class? Or better said, why modify only this class and not others?

Reply:
Sharp!
With the implementation of Dropwizard, metric registry is the storage of metric objects, and exporters are around the registry to send metric object value to outside.
In this test, which only concerns the storage of metrics, it is ok to not start (binding exporter to register).

With the implementation of Micrometer, there's no concept of exporter. Registry is for both storage and exporting. To solve the multiple exporting problem, Micrometer introduced a compositeRegistry, which is a API wrapper. CompositeRegistry must bind a concrete registry, such as Telegraf, InfluxDB, to land metric object into a concrete place.
Without binding to concrete registry, the operation to metric object is just an no-op.
Seems only this test class didn't call start() (where the binding happens), so I have to add it; other test classes already do this.